README.md

```diff
@@ -92,3 +94,9 @@ In order for your translation to appear on this doc, you need to:
 
 1. Make sure that your translation has been reviewed by members of your local community on Angle Discord. We will not accept any translation that has not been checked by other community members.
 2. Submit a pull request to the `angle-docs` repository and respect the formatting and conventions already in place for the `russian` section of the docs.
+
+## [Side Products](side-products/README.md)
+
+Angle Labs launched and is currently maintaining Angle Protocol.
```

Comment:
Let's not write maintaining the protocol, it should be contributing, but since we already write that above, no need to put this sentence